test/fixtures/addressesUsingIdsQuery/no_layers_with_boosts.json

```diff
@@ -0,0 +1,63 @@
+{
+  "query": {
+    "function_score": {
+      "query": {
+        "bool": {
+          "minimum_number_should_match": 1,
+          "should": [
+            {
+              "bool": {
+                "_name": "fallback.address",
+                "boost": 17,
+                "must": [
+                  {
+                    "match_phrase": {
+                      "address_parts.number": "housenumber value"
+                    }
+                  },
+                  {
+                    "match_phrase": {
+                      "address_parts.street": "street value"
+                    }
+                  }
+                ],
+                "filter": {
+                  "term": {
+                    "layer": "address"
+                  }
+                }
+              }
+            },
+            {
+              "bool": {
+                "_name": "fallback.street",
+                "boost": 19,
+                "must": [
+                  {
+                    "match_phrase": {
+                      "address_parts.street": "street value"
+                    }
+                  }
+                ],
+                "filter": {
+                  "term": {
+                    "layer": "street"
+                  }
+                }
+              }
+            }
+          ],
+          "filter": {
+            "bool": {
+              "minimum_number_should_match": 1,
+              "must": []
+            }
```

Comment:
nit: if possible, we could omit conditions which contain no valid query properties. (this may be out-of-scope fo this PR).

also, in this case the property `minimum_number_should_match` has no target as no `should` conditions exist.